src/simd/distances_powerpc.cc

```diff
@@ -0,0 +1,540 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#if defined(__powerpc__)
+
+#include <altivec.h>   /* Required for the Power GCC built-ins  */
+
+#include "distances_powerpc.h"
+
+#include <cmath>
+
+#define FLOAT_VEC_SIZE 4
+#define INT32_VEC_SIZE 4
+#define INT8_VEC_SIZE  16
+
+namespace faiss {
+
+float
+fvec_L2sqr_ref_ppc(const float* x, const float* y, size_t d) {
```

Comment:
please call the function `fvec_L2sqr_ppc`, not `fvec_L2sqr_ref_ppc`. 
Same for all other functions.